template/conftest.py

```diff
@@ -16,19 +16,19 @@
 
 BACKENDS = ["speculos", "ledgercomm", "ledgerwallet"]
 
+DEVICES = ["nanos", "nanox", "nanosp", "all"]
```

Comment:
+ stax as well

Reply:
I think we should add when SDK + speculos is public and Stax code name is fixed.